DESCRIPTION

```diff
@@ -1,7 +1,7 @@
 Package: BayesMallows
 Type: Package
 Title: Bayesian Preference Learning with the Mallows Rank Model
-Version: 1.0.4.9000
+Version: 1.0.4.9001
```

Comment:
Good morning! Any plans to bump the version number up (e.g. to 1.1.0)?

Reply:
Yes, on CRAN release.

Reply:
I was thinking 1.1.0 on CRAN release, but the master branch on GitHub contains the development versions, so here we increase only to 9001 for now.